src/app/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,56 @@
+.sign-in-page-background {
+  height: 100%;
+  background: url("../../assets/auth-background.jpg") no-repeat;
+  background-size: cover;
+}
+
+.container {
+  max-width: 100rem;
+  padding: 0 1rem;
+  height: 100%;
+  margin-inline: auto;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.sign-in-form {
+  max-width: 32rem;
+  width: 100%;
+
+  &__title {
+    font-family: 'impact', sans-serif;
+    font-size: 2.625rem;
```

Comment:
not sure that picking font size in rem, is a good idea. at least from the point of having ugly numbers like 2.625

Reply:
using rem's might be useful if you want to increase the font-size in the whole app just changing root font-size. but using em's isn't a good idea, see why
https://www.youtube.com/watch?v=pautqDqa54I&t=201s

to hide ugly numbers i guess we can use the following approach

![image](https://user-images.githubusercontent.com/48721238/232462082-49ffbce9-e519-44ca-ac9b-accbbaf85610.png)